README.md

```diff
@@ -56,6 +56,31 @@ const results = await conn.execute('select 1 from dual')
 console.log(results)
 ```
 
+### Transactions
+
+Use the `transaction` function to safely perform database transactions. If any unhandled errors are thrown during execution of the transaction, the transaction will be rolled back.
+
+The following example is based on [the Slotted Counter Pattern](https://planetscale.com/blog/the-slotted-counter-pattern).
+
+```ts
+import { connect } from '@planetscale/database'
+
+const config = {
+  host: '<host>',
+  username: '<user>',
+  password: '<password>'
+}
+
+const conn = connect(config)
+const results = await conn.transaction((tx) => {
+  return Promise.all([
+    tx.execute('INSERT INTO branches (database_id, name) VALUES (?, ?), [42, "planetscale"]'),
```

Comment:
I'm not sure what this query is doing, but, this seems incorrect.

Reply:
Both of these seem like you're trying to pass in arguments.

```ts
tx.execute('INSERT INTO branches (database_id, name) VALUES (?, ?)', [42, "planetscale"]),
tx.execute('INSERT INTO slotted_counters(record_type, record_id, slot, count) VALUES (?, ?, RAND() * 100, 1) ON DUPLICATE KEY UPDATE count = count + 1', [branch_count, database_id])
```

? I think is what you're intending to do.

Reply:
Good catch, thank you!